themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
This template appears to duplicate a lot of logic from the AbstractBase version while adding some EDS-specific details. Might there be a way to use sub-templates, view helpers, and/or new record driver methods to reduce the amount of duplication and make long-term maintenance a little easier? Right now, I believe that we have managed to avoid platform-specific export templates, and I'm concerned that this might get forgotten or out of sync over time if we're not careful.